### PR TITLE
 Solve a problem of nmake build（use vs2017）

### PR DESCRIPTION
**Pull request by hxznhf**
Solve a problem of nmake build（use vs2017）

### CODE REVIEW

**Comment by kjellahl** [comment]
There are a number of problems with this PR.
- You don't explain which problem it solves.
- One of the 3 commits has no commit message.
- One of the commits adds a file with Chinese characters both in its name and its contents.
- You modify the .pc.in in sigc++config.h.in files in such a way that your
  modifications are hard to follow. Are the .pc files used by MSVC? The are made for
  the pkg-config command in Linux systems.

Your PR needs rework.

@fanc999 Chun-wei, since this concerns NMake, can you comment? However, I would not
mind if you wait until the reporter has explained what's the problem.

**Comment by fanc999** [comment]
Hi,

I am also having trouble trying to understand the issue that this attempts to solve...

Note that the files that were updated in this PR should have the proper entries in the release source tarball, so I don't think the changes are needed for this regard.


With blessings, and cheers!

**Comment by fanc999** [comment]
Oh, also... 

Is there a particular need for you to modify the .pc.in file, since Visual Studio builds do not generally use pkg-config files, unless when used with build systems like Meson.

**Comment by murraycu** [comment]
> Note that the files that were updated in this PR should have the proper entries in the release source tarball

You might like to mention that in the new building section in the README.txt. Thanks.

**Comment by kjellahl** [comment]
@hxznhf Please explain which problem your PR solves. You can for instance include
error messages from VS2017. If it's not obvious from the error messages, you also
need to explain why you need to modify files that MSVS usually does not use.

@murraycu
>You might like to mention that in the new building section in the README.md.

I suppose this comment is meant for Chun-wei Fan, but I don't understand what
you want mentioned. Shall he mention that all files that this PR modifies are,
as far as he knows, correct in the latest tarball?

IMO this PR shall be closed (not merged) unless the reporter gives more information
within a reasonable time. Do you want it to stay open until README.md has been
updated?

**Comment by kjellahl** [comment]
Closing this PR without merging. The reporter has not replied to requests for more information.
Feel free to reopen, if you can supply the requested information.
